src/Input.tsx

```diff
@@ -15,7 +15,7 @@ const AutosizeWrapper: React.FunctionComponent<Partial<{ className: string }>> =
   <AutosizeInput {...props} inputClassName={className} placeholderIsMinWidth />
 );
 
-export const Input: React.FunctionComponent<IInput> = props => {
+export const Input = React.forwardRef<HTMLOrSVGElement, IInput>((props, ref) => {
```

Comment:
```suggestion
export const Input = React.forwardRef<HTMLInputElement, IInput>((props, ref) => {
```

Reply:
We can't actually do this one right? `as` is an exposed prop for `Input`.

Reply:
Yeah, but this is wrong - let's no expose it. Input must be input. What else could it be? Table, paragraph?

Reply:
Textarea maybe? But yeah... I'm good with removing the 'as' prop here.

Reply:
We do have a separate component for Textarea. :P